docs/examples.rst

```diff
@@ -263,11 +263,11 @@ CybOX type.
 
     <cyboxCommon:HashListType>
         <cyboxCommon:Hash>
-            <cyboxCommon:Type xsi:type="cyboxVocabs:HashNameVocab-1.0">MD5</cyboxCommon:Type>
+            <cyboxCommon:Type>MD5</cyboxCommon:Type>
```

Comment:
I changed it because the dictionaries used to generate the XML do not include an xsi:type key. The doctests were failing because of that, so I took out the `xsi:type` stuff rather than add it to the dictionary. It's still valid :) I can see what happens when I add `xsi:type` to the dictionary.